_data-prepper/pipelines/expression-syntax.md

```diff
@@ -219,47 +174,15 @@ White space is **required** surrounding set initializers, priority expressions,
 | `and`, `or`, `not`   | Conditional operators    | Yes                  | `/a<300 and /b>200`                                            | `/b<300and/b>200`                     |
 | `,`                  | Set value delimiter      | No                   | `/a in {200, 202}`<br>`/a in {200,202}`<br>`/a in {200 , 202}` | `/a in {200,}`                        |
 
-
 ## Functions
 
 Data Prepper supports the following built-in functions that can be used in an expression.
 
-### `length()`
```

Comment:
I think we should keep this format because of the complexity.

And honestly, I'd like to move toward using sub-pages for functions, similar to what we do for processors.

```
Functions
  cidrContains
  contains
  getMetadata
  hasTags
  join
  length
```

Reply:
Do we plan on adding additional functions at @dlvenable? I originally had them as section in the functions page so that they were link-able, however, I'm not sure there's enough info on each individual function to justify sub-pages, unless we plan on adding more information to each.

Reply:
Moved to subpages.

Reply:
@Naarcha-AWS , Yes, this is something we plan to expand upon and there may be many more.

@vagimeli , Thank you for making sub-pages!